--- origsrc/check-0.9.6/src/Makefile.am	2008-12-29 07:48:45.000000000 -0600
+++ src/check-0.9.6/src/Makefile.am	2009-03-18 03:12:37.451852500 -0500
@@ -38,7 +38,7 @@ $(EXPORT_SYM): check.h.in
 	sed -n -e 's/^..*CK_EXPORT[[:space:]][[:space:]]*\([[:alnum:]_][[:alnum:]_]*\)..*$$/\1/p' @top_srcdir@/src/check.h.in > $@
 
 libcheck_la_DEPENDENCIES= $(EXPORT_SYM)
-libcheck_la_LDFLAGS	= -export-symbols $(EXPORT_SYM)
+libcheck_la_LDFLAGS	= -export-symbols $(EXPORT_SYM) -no-undefined
 libcheck_la_SOURCES	= $(CFILES) $(HFILES)
 libcheck_la_LIBADD	= @GCOV_LIBS@
 
