diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x Makefile.in.in -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x mdate-sh -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x omf.make -x xmldocs.make origsrc/check-0.9.5/Makefile.am src/check-0.9.5/Makefile.am
--- origsrc/check-0.9.5/Makefile.am	2006-11-21 17:19:11.000000000 -0600
+++ src/check-0.9.5/Makefile.am	2006-11-21 21:42:37.429500000 -0600
@@ -19,7 +19,6 @@
 EXTRA_DIST = SVNChangeLog ChangeLogOld check.pc.in $(m4data_DATA)
 
 ## install docs
-docdir = $(datadir)/doc/$(PACKAGE)
 doc_DATA = SVNChangeLog ChangeLog ChangeLogOld NEWS README COPYING.LESSER
 
 ## install check.m4 with AM_PATH_CHECK in it
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x Makefile.in.in -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x mdate-sh -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x omf.make -x xmldocs.make origsrc/check-0.9.5/configure.ac src/check-0.9.5/configure.ac
--- origsrc/check-0.9.5/configure.ac	2006-11-21 17:19:11.000000000 -0600
+++ src/check-0.9.5/configure.ac	2006-11-21 21:43:30.445125000 -0600
@@ -2,7 +2,7 @@
 # Process this file with autoconf to produce a configure script.
 
 # Prelude.
-AC_PREREQ([2.59])
+AC_PREREQ([2.60])
 AC_INIT([Check], [0.9.5], [check-devel@lists.sourceforge.net])
 
 # unique source file --- primitive safety check 
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x Makefile.in.in -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x mdate-sh -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x omf.make -x xmldocs.make origsrc/check-0.9.5/doc/Makefile.am src/check-0.9.5/doc/Makefile.am
--- origsrc/check-0.9.5/doc/Makefile.am	2006-11-21 17:19:13.000000000 -0600
+++ src/check-0.9.5/doc/Makefile.am	2006-11-21 21:41:42.820125000 -0600
@@ -106,8 +106,6 @@
 
 ## what to install
 
-docdir = $(datadir)/doc/$(PACKAGE)
-
 # install money example
 
 exampledir = $(docdir)/example
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x Makefile.in.in -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x mdate-sh -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x omf.make -x xmldocs.make origsrc/check-0.9.5/src/Makefile.am src/check-0.9.5/src/Makefile.am
--- origsrc/check-0.9.5/src/Makefile.am	2006-11-21 17:19:13.000000000 -0600
+++ src/check-0.9.5/src/Makefile.am	2006-11-21 21:30:48.208125000 -0600
@@ -8,6 +8,8 @@
 
 libcheck_la_LIBADD=@GCOV_LIBS@
 
+libcheck_la_LDFLAGS=-no-undefined
+
 libcheck_la_SOURCES=\
 	check.h		\
 	check.c		\
